### PR TITLE
Adding clarification and more explicit ADFS instructions

### PR DESCRIPTION
**Pull request by jakubvul**
As per https://github.com/stormpath/stormpath-documentation/issues/324

### CODE REVIEW

File: source/appendix-saml.rst [hunk 1]
```diff
@@ -1017,7 +1017,11 @@ By default, the only user information that is passed by ADFS is the User Princip
 
 #. Back in the Stormpath Admin Console, on your ADFS Directory page, find the "SAML Configuration" section, and go to the "Attribute Mappings" tab.
 
-#. Here you can specify which of the ADFS Claims you would like to map to which Stormpath Account attribute. For example, if you mapped the LDAP Attribute "Given-Name" to the ADFS Claim "firstName", then you would put the "Attribute Name" as "firstName" and the "Stormpath Field Name" as "surname".
+#. Here you can specify which of the ADFS Claims you would like to map to which Stormpath Account attribute. For example, if you mapped the LDAP Attribute ``Given-Name`` to the ADFS Claim ``firstName``, then you would put the "Attribute Name" as ``firstName`` and the "Stormpath Field Name" as ``surname``.
```
Comment (lhazlewood) [inline source/appendix-saml.rst:1020]: instead of `surname` don't you mean `givenName` ?

File: source/appendix-saml.rst [hunk 2]
```diff
@@ -1017,7 +1017,11 @@ By default, the only user information that is passed by ADFS is the User Princip
...
+
+.. note::
+
+  You can achieve this clicking on "Add Rule" and selecting "Send LDAP Attributes as Claims". Next, select "Active Directory" as your attribute store. In the mapping table, you can add your rules. Following the above example, if one of your AD/LDAP attributes is ``Given-Name``, you can map that to Stormpath's ``firstName`` attribute. When your users first log in with their Active Directory credentials and mirrored Accounts are created for them in Stormpath, you should see the ``firstName`` attribute matching their ``given-name`` in Active Directory.
```
Comment (lhazlewood) [inline source/appendix-saml.rst:1024]: we don't have a `firstName` attribute - it is `givenName` ?

Comment (mdeggies) [reply source/appendix-saml.rst:1024]: yep good catch!

File: source/appendix-saml.rst (hunk 1 above)
Comment (mdeggies) [reply source/appendix-saml.rst:1020]: nice catch again!! surname should be changed to givenName.

**Pull request merged**